src/main/java/nl/tudelft/contextproject/model/level/util/CorridorExponentialExtension.java

```diff
@@ -0,0 +1,269 @@
+package nl.tudelft.contextproject.model.level.util;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import nl.tudelft.contextproject.util.RandomUtil;
+
+/**
+ * Utility class that contains methods which expand corridors using an exponential distribution.
+ */
+public final class CorridorExponentialExtension {
+	
+	/**
+	 * Minimum extension of corridors.
```

Comment:
The pull request description actually gives a more detailed description than this comment. Perhaps you could copy it from the PR?